### DISCUSSION TITLE
2008/05/21/%E6%8B%94%E6%8E%89%E4%BA%86%E6%9C%80%E5%90%8E%E4%B8%80%E9%A2%97%E6%99%BA%E9%BD%BF

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#219] [Announcements]
# 2008/05/21/%E6%8B%94%E6%8E%89%E4%BA%86%E6%9C%80%E5%90%8E%E4%B8%80%E9%A2%97%E6%99%BA%E9%BD%BF

我总共就长了三颗智齿，都没长好。前两颗早就被拔掉了，第三颗在左上，情况稍好一点，但是也有点向下颌骨和颅骨结合的部位倾斜。并且它还比别的牙齿多长出来一节，有时会挡住下牙的活动。所以这次下定决心把它拔掉。

https://blog.qizhen.xyz/2008/05/21/%E6%8B%94%E6%8E%89%E4%BA%86%E6%9C%80%E5%90%8E%E4%B8%80%E9%A2%97%E6%99%BA%E9%BD%BF

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Yunjie
2009年08月26日 14:02 编辑
我也在想要不要去拔，害怕。。。

回复
茂
2008年05月22日 11:03 编辑
拔阻生智齿很凶险的，很容易伤到神经，要小心哦！

回复
Yan
2008年05月21日 22:10 编辑
 
我的第二颗智齿在墨尔本永久定居了　正在考虑第三颗移民到哪儿呢
 
除了眼结石　去年又长了个肾结石　集体向你的腮帮子问好~

回复
yoyo
2008年05月21日 21:22 编辑
到现在我一颗智齿没长 奇怪
更奇怪的是 你咋老长写奇奇怪怪的毛病啊
 

回复
HA
2008年05月21日 19:21 编辑
偶也有三颗智齿~发炎起来疼得要命；但就是不敢去拔，害怕~

回复
